src/khoj/routers/api.py

```diff
@@ -70,8 +70,35 @@ async def set_content_config_github_data(updated_config: GithubContentConfig):
         except Exception as e:
             return {"status": "error", "message": str(e)}
 
+    @api.post("/delete/config/data/content_type/{content_type}", status_code=200)
+    async def remove_content_config_data(content_type: str):
+        if not state.config or not state.config.content_type:
+            return {"status": "ok"}
+
+        if state.config.content_type:
+            state.config.content_type[content_type] = None
+
+        try:
+            save_config_to_file_updated_state()
+            return {"status": "ok"}
+        except Exception as e:
+            return {"status": "error", "message": str(e)}
+
+    @api.post("/delete/config/data/processor/conversation", status_code=200)
+    async def remove_processor_conversation_config_data():
+        if not state.config or not state.config.processor or not state.config.processor.conversation:
+            return {"status": "ok"}
+
+        state.config.processor.conversation = None
+
+        try:
+            save_config_to_file_updated_state()
+            return {"status": "ok"}
+        except Exception as e:
+            return {"status": "error", "message": str(e)}
```

Comment:
Why do the 2 new delete methods not raise an HTTP exception?

Reply:
This is how all the API endpoints are being handled right now, so I was maintaining the pattern for consistency. I do think it makes sense to go through and update/standardize them to use HTTP exceptions.